contrib/kafka/pkg/dispatcher/dispatcher.go

```diff
@@ -38,7 +38,7 @@ type KafkaDispatcher struct {
 	updateLock sync.Mutex
 
 	receiver   *provisioners.MessageReceiver
-	dispatcher *provisioners.MessageDispatcher
+	dispatcher provisioners.Dispatcher
```

Comment:
if you have a `MessageReceiver`, I would prefer to have a `MessageDispatcher`.

Reply:
Plus it does not dispatch provisioners...

Reply:
Well, insofar it compiles and passes the tests, which means provisioner is not required.
Question is then: why does even Dispatcher interface exist?